python/main.py

```diff
@@ -0,0 +1,70 @@
+from dotenv import load_dotenv
+import os
+import redis
+
+
+load_dotenv()
+
+REDIS_URL = os.environ.get("REDIS_URL", "localhost")
+REDIS_SET_KEY = "mobydick:words:set"
+REDIS_BLOOM_KEY = "mobydick:words:bloom"
+REDIS_HLL_KEY = "mobydick:words:hyperloglog"
+REDIS_TOPK_KEY = "mobydick:words:topk"
+
+# Create a client and connect to Redis
+client = redis.Redis(REDIS_URL)
+
+
+# Clean up from any previous run...
+
+client.delete(REDIS_SET_KEY)
+client.delete(REDIS_BLOOM_KEY)
+client.delete(REDIS_HLL_KEY)
+client.delete(REDIS_TOPK_KEY)
+
+# Initialize Bloom Filter and Top K.
+
+bloom_filter = client.bf()
+bloom_filter.create(REDIS_BLOOM_KEY, 0.01, 20000)
+
+top_k = client.topk()
+top_k.reserve(REDIS_TOPK_KEY, 10, 8, 7, 0.9)
+
+# Process the file of words.
+
+with open("../moby_dick_just_words.txt", "r") as fi:
+    for line in fi:
+        for raw_word in line.split():
+            word = raw_word.strip().lower()
+            bloom_filter.add(REDIS_BLOOM_KEY, word)
```

Comment:
This takes a very long time to run because it's not using a pipeline, like the Node code does.  Please refactor to use a pipeline https://redis.readthedocs.io/en/stable/examples/pipeline_examples.html

Reply:
I am working on a fix. Should be done in a few hours.